test/gurka/test_car_restrictions.cc

```diff
@@ -0,0 +1,69 @@
+#include "gurka.h"
+#include <gtest/gtest.h>
+
+using namespace valhalla;
+
+class CarRestrictionTest : public ::testing::TestWithParam<std::pair<std::string, std::string>> {
```

Comment:
can you put that with `gurka/test_truck_restrictions` and maybe rename that? recently we're trying not to create any more tests unless it's really warranted.